package.json

```diff
@@ -23,14 +25,26 @@
     "isomorphic-fetch": "^2.2.1",
     "joi-browser": "^10.0.6",
     "node-fetch": "^1.6.3",
```

Comment:
Unrelated, but do we need **node-fetch** still if we're using **isomorphic-fetch**?